servers/partner-ci/common.yaml

```diff
@@ -129,6 +129,93 @@
             WORKSTATION_USERNAME=vmware;
 
 - parameter:
+    name: common-fuel-parameters
```

Comment:
We need to avoid of passing unused parameters.
We should facilitate "clear fuel" jobs.